businesscentral/image.yaml

```diff
@@ -1,13 +1,13 @@
 name: "jboss-bpmsuite-7/bpmsuite70-businesscentral"
 description: "Red Hat JBoss BPM Suite 7.0 Business Central container image"
-version: "7.0.0"
+version: "7.0.0.CAP02"
 from: "jboss-eap-7/eap70:latest"
 user: 185
 envs:
     - name: "JBOSS_PRODUCT"
-      value: "bpmsuite"
-    - name: "JBOSS_BPMSUITE_VERSION"
-      value: "7.0.0"
+      value: "bpmsuite-businesscentral"
+    - name: "JBOSS_BPMSUITE_BUSINESSCENTRAL_VERSION"
+      value: "7.0.0.CAP02"
```

Comment:
This should be `7.0.0`. Besides this you need to update all envs, not just one of them, if you do so.